robot/basestation/static/js/elements/feed/navbar.js

```diff
@@ -86,98 +111,28 @@ $(document).ready(() => {
   }
 
   // select mux channel using mux_select service
-  $('#mux-0').mouseup(function () {
-    // Rover
-    if (isListenerOpen() && getCookie('serialType') == 'uart') {
-      appendToConsole('Don\'t change the mux channel while a listener is open!')
-    } else {
-      requestMuxChannel('#mux-0', function (msgs) {
-        printErrToConsole(msgs)
-
-        if (msgs[0] == true && window.location.pathname == '/rover') {
-          console.log('Activating Rover Listener Node')
-
-          let serialType = getCookie('serialType')
-
-          if (serialType == '') {
-            appendToConsole('Serial type not yet defined!')
-            return
-          }
-
-          // automating opening listener and sending MCU ping in UART mode
-          if (serialType == 'uart') {
-            requestTask(
-              ROVER_LISTENER_TASK,
-              STATUS_START,
-              function (msgs) {
-                if (msgs[0]) {
-                  $('#toggle-rover-listener-btn')[0].checked = true
-                  // try pinging MCU
-                  wait(1000)
-                  sendRequest('Rover', 'ping', printErrToConsole)
-                } else {
-                  $('#toggle-rover-listener-btn')[0].checked = false
-                }
-              },
-              serialType
-            )
-          }
-        }
-      })
+  $("a[id^='mux-']").mouseup(e => {
+    console.log($(e.target).attr("id"))
+    id = $(e.target).attr("id")
+    if (id != 'mux-3') {
```

Comment:
what happens to the code from `mux-3`?

Reply:
There's a separate that handles the changing the device channel to mux-3. So I just made it so that it runs the special function for when the mux-3 button is clicked, instead of running the function that handles switching to the other channels. Although, I just noticed that there's a cleaner way of handling this, so I'm going to make that change real quick

Reply:
I refactored those two functions into one, making the code a bit neater